### PR TITLE
Replace deprecated rmrf_enter

### PR DESCRIPTION
**Pull request by giorgiosironi**
For https://github.com/elifesciences/issues/issues/4986

### CODE REVIEW

**Comment by giorgiosironi** [comment]
We should fare well for a while even without this, took weeks for it to fill the disk.

File: salt/elife-bot/init.sls [hunk 1]
```diff
@@ -115,22 +115,15 @@ elife-bot-digest-cfg:
...
-        - template: jinja
-        - makedirs: True
-        - require:
-            - global-python-requisites
 
     # 2am, every day
     cron.present:
         - identifier: temp-files-cleaner
-        - name: python /opt/rmrf_enter/elife-bot.py > /dev/null
+        - name: find . -maxdepth 1 -type d -name '20*' -mtime +{{ pillar.elife_bot.rmrf_enter.days }} -exec rm -r '{}' \; >> /tmp/elife-bot-temporary-files-cleaner.log
```
Comment (gnott) [inline salt/elife-bot/init.sls]: Please excuse if I don't fully understand how `rmft_enter` was used, but here I assume the `find .` this is already in the `/opt/elife-bot/tmp` or `/bot-tmp` folder by this point?

Comment (giorgiosironi) [reply salt/elife-bot/init.sls]: Yeah this lacks a working directory, I'll substitute `.`

Comment (lsh-0) [reply salt/elife-bot/init.sls]: `rmrf_enter` was/is a library that was installed on all machines and made mass deletions of files a little bit safer. It was written before bash became my go-to for this sort of thing and at the end of the day, while it worked, it wasn't maintained and it wasn't particularly worth the extra complexity that comes from maintaining a library and and per-project scripts that use that library. It's usage got whittled away to just elife-bot and now we're removing that as well.

Comment (lsh-0) [reply salt/elife-bot/init.sls]: fixed up that working directory

**Comment by lsh-0** [comment]
The file being appended to in the log will grow over time. It's not serious but it is another thing that slowly accumulates.

**Comment by giorgiosironi** [comment]
> The file being appended to in the log will grow over time.

It's an half-hearted attempt to capture errors, in theory the command should have no output if successful (but not tested it in all error cases).

**Comment by lsh-0** [comment]
It's fine, really. When it (eventually) becomes a problem the fix will be dead simple and won't bother us again for a few years: `echo >  /tmp/elife-bot-temporary-files-cleaner.log`

**Pull request merged**